assets/src/components/propertiesPanel/vehiclePropertiesPanel.tsx

```diff
@@ -198,15 +199,25 @@ const StatusContent = ({ selectedVehicle }: { selectedVehicle: Vehicle }) => (
   </>
 )
 
-const VehiclePropertiesPanel = ({ selectedVehicle }: Props) => {
-  const [tabMode, setTabMode] = useState<TabMode>("status")
+const VehiclePropertiesPanel = ({
+  selectedVehicle,
+  initialTab = "status",
+}: Props) => {
+  const [tabMode, setTabMode] = useState<TabMode>(initialTab)
 
   return (
     <div className="c-vehicle-properties-panel">
       <Header
         vehicle={selectedVehicle}
         tabMode={tabMode}
-        setTabMode={setTabMode}
+        setTabMode={(tab) => {
+          if (tab !== tabMode) {
+            window.FS?.event("Switched tab in Vehicle Properties Panel", {
+              tab_str: tab,
+            })
```

Comment:
I know we had discussed changing our DOM representation to make it easier to name elements in FS, but I opted to use an FS event for two reasons
- Joan had mentioned that in the past FS events had been easier to use for conversion tracking, so I wanted to be compliant for that use case.
- The ticket specifically states a FS event
	- We haven't had a chance to re-review this ticket since the discussion yesterday, so this could be a reason to hold off.